internal/evaluator/builder.go

```diff
@@ -274,7 +274,7 @@ func (bldr *Builder) processFieldExpressions(
 	var opts []cel.EnvOption
 	if fieldDesc.Kind() == protoreflect.MessageKind {
 		opts = []cel.EnvOption{
-			cel.Types(dynamicpb.NewMessage(fieldDesc.Message())),
+			cel.Types(dynamicpb.NewMessage(fieldDesc.ContainingMessage())),
```

Comment:
Let's add both in this case.

Reply:
resolved in [5a9fae1](https://github.com/bufbuild/protovalidate-go/pull/65/commits/5a9fae13ee69efa176b7471a672e09dfb1624810)